frontend/src/metabase-lib/expressions/suggest.unit.spec.ts

```diff
@@ -61,6 +78,8 @@ const FIELDS_CUSTOM_NON_NUMERIC = [
   { type: "fields", text: "[text] " },
 ];
 
+const DEFAULT_QUERY = DEFAULT_EXPRESSION_QUERY as DatasetQuery;
```

Comment:
It's trivial to convert `__support__/expressions.js` to TS and then we wouldn't need this cast.

Reply:
I had an idea to unify this file with another as we have 2 files with fixtures. Good point

Reply:
done https://github.com/metabase/metabase/pull/36265